joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -441,3 +441,121 @@ TEST_F(TestTrajectoryController, correct_initialization_using_parameters) {
   ASSERT_EQ(State::PRIMARY_STATE_INACTIVE, state.id());
   executor.cancel();
 }
+
+
+TEST_F(TestTrajectoryController, test_state_publish_rate) {
+  auto traj_controller = std::make_shared<joint_trajectory_controller::JointTrajectoryController>();
+  auto ret = traj_controller->init(test_robot, controller_name);
+  if (ret != controller_interface::CONTROLLER_INTERFACE_RET_SUCCESS) {
+    FAIL();
+  }
+
+  // fill in some data so we wont fail
+  auto traj_lifecycle_node = traj_controller->get_lifecycle_node();
```

Comment:
Consider having you're own test fixture where you do all this setup so you don't have duplicate code.

Reply:
Fixed in [505f8a2](https://github.com/ros-controls/ros2_controllers/pull/25/commits/505f8a2a6d1d985582501bc9492236ae8805e710)